src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -294,7 +260,6 @@ export const ConfirmSummary = ({
         )}
         <WithLazyMount
           shouldUse={Boolean(receiveAddress) && shouldForceManualAddressEntry === false}
-          shouldForceManualAddressEntry={shouldForceManualAddressEntry}
```

Comment:
I think we should add this back for safety. If ever the `ReceiveAddress` component shows, and then we end up in a manual address entry situation, `ReceiveAddress` will still be shown alongside it. While the way we are handling smart contract addresses only containing one account being evm may negate this, I think it still serves the intended purpose in case.

![image](https://github.com/user-attachments/assets/5f2afb11-1f4a-4f50-bcc4-b330c8ffcba5)

Reply:
Brought back in d19e28686cc9cdc70fa58441aa0cdd8fb6c0a45b :

- Native buy asset THOR swap with sc receive address

<img width="943" alt="image" src="https://github.com/user-attachments/assets/b638e9fa-443a-4955-a961-c1063ed77124">

- sc sell address, native buy asset THOR swap to an EOA receive address

<img width="941" alt="image" src="https://github.com/user-attachments/assets/36cef879-14f5-4fb0-bf64-0fa6c3527e82">

- sc sell address, token buy asset THOR swap to a sc receive address

<img width="953" alt="image" src="https://github.com/user-attachments/assets/9cfe23d8-59ac-4cff-b3cb-a3e2e417cdee">

- EOA <-> EOA, native asset <-> native asset THOR swap

<img width="969" alt="image" src="https://github.com/user-attachments/assets/b4ec1cd7-e794-4e40-9341-a5e031306bbd">